commit 98b33905c89b7e5c309c74ae32302a5745901a6e
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Nov 18 14:54:10 2018 +0100

    random: Do not require the latest gcrypt version, just use a version known to work
    
    This helps with dependency management of binary packages.
    
    * trans/random.c (pool_initialize): Pass "1.8.0" to gcry_check_version
    instead of GCRYPT_VERSION.

diff --git a/trans/random.c b/trans/random.c
index 76b780bc..f0b99dd4 100644
--- a/trans/random.c
+++ b/trans/random.c
@@ -62,7 +62,7 @@ pool_initialize (void)
   error_t err;
   gcry_error_t cerr;
 
-  if (! gcry_check_version (GCRYPT_VERSION))
+  if (! gcry_check_version ("1.8.0"))
     error (1, 0, "libgcrypt version mismatch\n");
 
   cerr = gcry_control (GCRYCTL_INITIALIZATION_FINISHED, 0);
